lib/Service/Appointments/MailService.php

```diff
@@ -272,8 +275,10 @@ private function getSysEmail(): string {
 	}
 
 	public function sendOrganizerBookingInformationEmail(Booking $booking, AppointmentConfig $config, string $calendar) {
+		
+		$userId = $config->getUserId();
 		/** @var IUser $user */
-		$user = $this->userManager->get($config->getUserId());
+		$user = $this->userManager->get($userId);
```

Comment:
Same as above.

We can probably get rid of the user object completely though. It only seems to be used to check if the user actually exists some lines further down.